### ISSUE TITLE
Allow floor/round etc of numbers

### ISSUE THREAD

**Issue by GreenImp** [GreenImp/rpg-dice-roller#64]
Investigate the possibility of allowing numbers to be rounded in various different ways.

**Comment by GreenImp** [comment]
If we integrate [Math.js](https://mathjs.org/) as part of #34, that may possibly resolve this one. It has a lot of built in functions for formula parsing. If there aren;t any built in version of round etc. then we can easily imlement them.

**Comment by GreenImp** [comment]
Resolved in pull request #76

You can now use various mathematical functions in notations, including ones for rounding.

Some examples:

```
round(4d10/3): round([3,6,1,1]/3) = 3.7  // round the result to the nearest whole number
floor(4d10/3): round([3,6,1,1]/3) = 3.6  // round the result down to the nearest whole number
ceil(4d10/3): round([3,6,1,1]/3) = 3.7   // round the result up to the nearest whole number
```
